api/models/templates/app.tmpl

```diff
@@ -498,6 +498,28 @@
                       ] ] }
                     }
                   }
+                },
+                {
+                  "Effect": "Allow",
+                  "Action": "logs:CreateLogGroup",
```

Comment:
honestly it might not be, but I tried a variety of subsets and nothing was successful until I straight up copied the AWS template `AWSLambdaBasicExecutionRole`